packages/blade/src/components/Input/TextArea/TextArea.tsx

```diff
@@ -26,6 +26,7 @@ type TextAreaProps = Pick<
   | 'onChange'
   | 'onFocus'
   | 'onBlur'
+  | 'onSubmit'
```

Comment:
Do we not need this on OTPInput, SelectInput?

Reply:
Not that I can think of. OTPInput has `onOTPFilled` which is called when all input fields are filled and it won't need onSubmit. SelectInput, I can't think of a use-case where onSubmit would be needed. We can always add it when use-case comes up